ngraph/test/runtime/ie/CMakeLists.txt

```diff
@@ -41,8 +41,3 @@ add_dependencies(ie_backend inference_engine)
 target_compile_definitions(ie_backend PRIVATE IE_BACKEND_DLL_EXPORTS)
 target_include_directories(ie_backend PUBLIC ${IE_MAIN_SOURCE_DIR}/include)
 target_link_libraries(ie_backend PUBLIC ngraph_backend inference_engine)
-
-install(TARGETS ie_backend
```

Comment:
Are all nGraph's unit tests still working with this change? The same question about the `interpreter_backend` target modification below.

Reply:
We don't have nGraph unit-tests in the OpenVINO package. I think it is why we have these changes,

Reply:
Ok, now I can see that the targets installation is removed. In this case it should be ok.